public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -664,3 +671,32 @@ function MaybeQueryLibrarySaveButton(props: { query: DataQuery }) {
   const { renderSaveQueryButton } = useQueryLibraryContext();
   return renderSaveQueryButton(props.query);
 }
+
+function AdaptiveTelemetryQueryActions({ query }: { query: DataQuery }) {
+  const extensionPointId = 'grafana/adaptivetelemetry/query/action';
```

Comment:
What about renaming it to `grafana/adaptivetelemetry/query/action/v1`. Also, could be a good idea to add it to the `PluginExtensionPoints` enum exposed from `@grafana/data`.